src/screens/BB-BUS/IndexScreen.js

```diff
@@ -98,35 +92,21 @@ export const IndexScreen = ({ navigation }) => {
     (data, areaId) => [(data[0].object || data[0]).id, data.length, areaId].join('-')
   );
 
-  const getCommunities = (data) => {
-    const snake_caseData = data?.directus?.community?.data;
+  const getAreas = (data) => {
+    const areas = data?.area;
```

Comment:
i found out that some areas show up twice

![IMG_814225044D8C-1](https://user-images.githubusercontent.com/1942953/149384181-7a5412f9-525e-490c-ab39-5089fb3b1773.jpeg)

Reply:
these are duplicates due to the name occurring with two different ids on the side of the external endpoint:

```
{
        "externalId": "100008239",
        "name": "Niemegk"
},
{
        "externalId": "100025122",
        "name": "Niemegk"
},
```